db/migrate/20220509083131_change_first_aid_expires_at_to_first_aid_achieved_on.rb

```diff
@@ -1,5 +1,5 @@
 class ChangeFirstAidExpiresAtToFirstAidAchievedOn < ActiveRecord::Migration[6.1]
-  def change
-    rename_column :employees, :first_aid_expires_at, :first_aid_achieved_on
-  end
-end
+    def change
```

Comment:
Did you mean to add this?

Reply:
this migration? We have this in main, when we rebase this branch didn't come. I think that's why first_aid_expires_at was there instead achieved_on